dart_io_mini_samples/example/other_resources.md

```diff
@@ -1,27 +1,27 @@
-Read the [dart:io library API docs](https://api.dartlang.org/docs/channels/stable/latest/dart_io.html).
+Read the [dart:io library API docs]({{site.dart_api}}/dart-io/dart-io-library.html).
 
-Here are links to the commonly used classes and packages in these
-examples:
+Here are links to the commonly used classes and packages in these examples:
 
-* Files, Directories, and Symlinks
+* Using files, directories, and symlinks
```

Comment:
Using files -> Files

(to match the titles in the TOC)

Which makes me notice that the "Interacting with processes" and "Working with paths" sections (and the corresponding list items here) should probably be just "Processes" and "Paths", for consistency.